client/src/utilities/NavigationLayout/NavigationLayout.tsx

```diff
@@ -63,7 +63,7 @@ const navigationContents = [
   {
     label: 'TEASE',
     icon: IconAbacus,
-    link: '/tease/',
+    link: `${window.location.origin}/tease`,
```

Comment:
this will not work either because a relative path is required :/

Reply:
it will otherwise resolve into prompt.ase.cit.tum.de/management/http://prompt.ase.cit.tum.de/tease